pkg/jobrunaggregator/jobrunbigqueryloader/uploader.go

```diff
@@ -189,23 +193,30 @@ func (o *jobLoaderOptions) Run(ctx context.Context) error {
 	for jobRunID := range jobRunProcessingCh {
```

Comment:
30s skim, so apologies if it it's a non-sequitor, but where is this channel closed?  this loop will not exit until jobRunProcessingCh is closed.